PasswordValidator_UnitTests/SimpleValidator_UnitTests.cs

```diff
@@ -10,7 +12,14 @@ internal class SimpleValidator_UnitTests
 
         public SimpleValidator_UnitTests()
         {
-            _validator = new SimpleValidator();
+            var rules = new List<IRule>
```

Comment:
Should be mock objects - not just three - try a collection of one, try an empty collection, try null (it is a public class someone could do that)..